sdk/zksend/src/links/list-created-links.ts

```diff
@@ -0,0 +1,163 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { bcs } from '@mysten/sui.js/bcs';
+import type { SuiClient } from '@mysten/sui.js/client';
+import { SuiGraphQLClient } from '@mysten/sui.js/graphql';
+import { graphql } from '@mysten/sui.js/graphql/schemas/2024-01';
+import { fromB64, normalizeSuiAddress } from '@mysten/sui.js/utils';
+
+import { ZkSendLink } from './claim.js';
+import type { ZkBagContractOptions } from './zk-bag.js';
+import { MAINNET_CONTRACT_IDS } from './zk-bag.js';
+
+const ListCreatedLinksQuery = graphql(`
+	query listCreatedLinks($address: SuiAddress!, $function: String!, $cursor: String) {
+		transactionBlocks(
+			last: 10
+			before: $cursor
+			filter: { signAddress: $address, function: $function, kind: PROGRAMMABLE_TX }
+		) {
+			pageInfo {
+				startCursor
+				hasPreviousPage
+			}
+			nodes {
+				effects {
+					timestamp
+				}
+				digest
+				kind {
+					__typename
+					... on ProgrammableTransactionBlock {
+						inputs(first: 10) {
+							nodes {
+								__typename
+								... on Pure {
+									bytes
+								}
+							}
+						}
+						transactions(first: 10) {
```

Comment:
Why only the first 10?

Reply:
10 is currently the max, and 5 is the default, so this is the most we can get right now